test/Dapr.Client.Test/StateApiTest.cs

```diff
@@ -839,8 +839,8 @@ public async Task TrySaveStateAsync_ValidateNonETagErrorThrowsException()
         {
             var client = new MockClient();
 
-            var response = client.CallStateApi<string>()
-            .Build();
+            await client.CallStateApi<string>()
+                .Build();
```

Comment:
Please put these two on one line.

Reply:
Thank you! Addressed in my latest commit!